python/py-rawpy/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-rawpy
+version             0.17.2
+revision            0
+license             MIT
+
+github.setup        letmaik rawpy 0.17.2 v
+github.tarball_from archive
+
+checksums           rmd160  9606f685559195220c4d0557644d85780db00144 \
+                    sha256  31d76329c231131b606bda0c0066ae69e0ed50839cd2ccd8c0bec278c1bb7a19 \
+                    size    67235904
+
+python.versions     310
+python.pep517       yes
+
+maintainers         {juliensalort.org:mp @jsalort} openmaintainer
+categories-append   graphics
+
+description         Python wrapper for the LibRaw library
+long_description    rawpy is an easy-to-use Python wrapper for the LibRaw library. It also contains some extra \
+                    functionality for finding and repairing hot/dead pixels.
+
+if {${name} ne ${subport}} {
+
+    patchfiles      setup.py.diff
+
+    depends_build-append \
+                    port:py${python.version}-cython \
+                    port:pkgconfig
+
+    depends_lib-append \
+                    port:libraw \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-scikit-image
+
+    depends_test-append \
+                    port:py${python.version}-pytest \
```

Comment:
the `pytest` can be removed here as the default with `test.run` (when using the `python` PG) is to set `python.test_framework pytest`